docs/JMX.md

```diff
@@ -0,0 +1,134 @@
+# Secure JMX
```

Comment:
Whom is this documentation targeted at? I'm somewhat worried this will just rot...

Reply:
added TODO to remove this once this exists in docs-site